docs/appendices/release-notes/unreleased.rst

```diff
@@ -73,12 +73,6 @@ Fixes
   :ref:`cursors <sql-fetch>`, even if the ``CURSOR`` was explicitly or
   automatically (session terminated) closed.
 
-- Added validation to reject inner column names containing special whitespace
```

Comment:
not sure we want to remove those in 5.2 - I think we had a similar situation before (https://github.com/crate/crate/pull/13044#discussion_r972070378). As far as I remember, we want to have all unreleased entries in 5.2 as we will release them in the upcoming 5.2.1

Reply:
ah, I see, so I shouldn't remove them from `master` as well, and do it with `5.2.1` instead, thx